cypress/e2e/tests/pages/fleet/advanced/bundles.spec.ts

```diff
@@ -1,15 +1,162 @@
+        expect(response?.statusCode).to.eq(201);
+        removeBundle = true;
+        bundlesNameList.push(customBundleName);
+      });
+      fleetBundles.waitForPage();
+      fleetBundles.bundlesList().rowWithName(customBundleName).checkVisible();
+    });
+
+    // Skipping until issue resolved: https://github.com/rancher/dashboard/issues/13990
+    // it('can Edit Config', () => {
+    //   cy.intercept('PUT', `/v1/fleet.cattle.io.bundles/${ defaultWorkspace }/${ customBundleName }`).as('editBundle');
+
+    //   fleetBundles.goTo();
+    //   fleetBundles.waitForPage();
+    //   fleetBundles.bundlesList().actionMenu(customBundleName).getMenuItem('Edit YAML').click();
+    //   fleetBundles.createBundlesForm(defaultWorkspace, customBundleName).waitForPage('mode=edit&as=yaml');
+    //   fleetBundles.createBundlesForm().title().invoke('text').then((title) => {
+    //     expect(title.replace(/\s+/g, ' ')).to.contain(`Bundle: ${ customBundleName }`);
+    //   });
+    //   fleetBundles.resourceDetail().resourceYaml().codeMirror().value()
+    //     .then((val) => {
+    //       // convert yaml into json to update values
+    //       const json: any = jsyaml.load(val);
+
+    //       json.metadata.namespace = localWorkspace;
+
+    //       fleetBundles.resourceDetail().resourceYaml().codeMirror().set(jsyaml.dump(json));
+    //     });
+    //   fleetBundles.resourceDetail().resourceYaml().saveOrCreate().click();
+    //   cy.wait('@editBundle').then(({ response }) => {
+    //     expect(response?.statusCode).to.eq(200);
+    //     expect(response?.body.metadata.namespace).equals(localWorkspace);
+    //   });
+    //   fleetBundles.waitForPage();
+    // });
+
+    it('can clone a bundle', () => {
+      cy.intercept('POST', '/v1/fleet.cattle.io.bundles').as('cloneBundle');
+
+      fleetBundles.goTo();
+      fleetBundles.waitForPage();
+      fleetBundles.bundlesList().actionMenu(customBundleName).getMenuItem('Clone').click();
+      fleetBundles.createBundlesForm(defaultWorkspace, customBundleName).waitForPage('mode=clone&as=yaml');
+      fleetBundles.createBundlesForm().title().invoke('text').then((title) => {
+        expect(title.replace(/\s+/g, ' ')).to.contain(`Bundle: Clone from ${ customBundleName }`);
+      });
+      fleetBundles.resourceDetail().resourceYaml().codeMirror().value()
+        .then((val) => {
+          // convert yaml into json to update values
+          const json: any = jsyaml.load(val);
+
+          json.metadata.name = `${ customBundleName }-clone`;
+
+          fleetBundles.resourceDetail().resourceYaml().codeMirror().set(jsyaml.dump(json));
+        });
+
+      fleetBundles.resourceDetail().resourceYaml().saveOrCreate().click();
+      cy.wait('@cloneBundle').then(({ response }) => {
+        expect(response?.statusCode).to.eq(201);
+      });
+      fleetBundles.waitForPage();
+      fleetBundles.bundlesList().rowWithName(`${ customBundleName }-clone`).checkVisible();
```

Comment:
If there's a resource count, this should match the total of the original bundle.